test-case/verify-kernel-boot-log.sh

```diff
@@ -114,8 +114,10 @@ dmic_switch_present()
 
     (set -x
      # name= is hardcoded in /usr/share/alsa/ucm2/*
+     # - https://github.com/thesofproject/sof-test/issues/1244
+     first_card_name=$(aplay -l | awk 'NR == 2 {print $3}')
```

Comment:
you could do the same with `arecord` and probably verify that the name starts with `sof`